### ISSUE TITLE
Angular 4 Issue

### ISSUE THREAD

**Issue by lisbuenas** [nekken/ng2-fullcalendar#41]
The last release of angular 4 has a running problem "Error: Uncaught (in promise): Error: Unexpected value 'CalendarComponent' declared by the module "

Current Angular Version used on project 4.1.3 and Angular CLI 1.1.0

**Comment by KrisBraun** [comment]
I don't think this package is using the correct approach by including the component directly rather than importing a module, because it's pulling in a separate dependancy on a specific version of Angular. This package uses proper module imports and works for me: https://github.com/lbertenasco/ap-ng2-fullcalendar
